dev-util/clion/clion-2025.1-r1.ebuild

```diff
@@ -0,0 +1,150 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop optfeature toolchain-funcs wrapper
+
+DESCRIPTION="A complete toolset for C and C++ development"
+HOMEPAGE="https://www.jetbrains.com/clion/"
+SRC_URI="
+	amd64? ( https://download.jetbrains.com/cpp/CLion-${PV}.tar.gz )
+	arm64? ( https://download.jetbrains.com/cpp/CLion-${PV}-aarch64.tar.gz )
+"
+
+LICENSE="|| ( IDEA IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal )
+	Apache-1.1 Apache-2.0 BSD BSD-2 CC0-1.0 CDDL-1.1 CPL-0.5 CPL-1.0
+	EPL-1.0 EPL-2.0 GPL-2 GPL-2-with-classpath-exception GPL-3 ISC JDOM
+	LGPL-2.1+ LGPL-3 MIT MPL-1.0 MPL-1.1 OFL-1.1 public-domain PSF-2
+	UoI-NCSA ZLIB"
+SLOT="0/2025"
+KEYWORDS="~amd64 ~arm64"
+RESTRICT="bindist mirror"
+
+BDEPEND="
+	dev-util/debugedit
+	dev-util/patchelf
+"
+
+RDEPEND="
+	>=app-accessibility/at-spi2-core-2.46.0:2
+	dev-libs/expat
+	dev-libs/glib:2
+	dev-util/lttng-ust:0/2.12
+	dev-libs/nspr
+	dev-libs/nss
+	dev-libs/wayland
+	dev-build/cmake
+	app-alternatives/ninja
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype:2
+	media-libs/mesa
+	net-print/cups
+	sys-apps/dbus
+	sys-libs/zlib
+	x11-libs/cairo
+	x11-libs/libdrm
+	x11-libs/libX11
+	x11-libs/libXcomposite
+	x11-libs/libXcursor
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXi
+	x11-libs/libXrandr
+	x11-libs/libXrender
+	x11-libs/libXtst
+	x11-libs/libXxf86vm
+	x11-libs/libxcb
+	x11-libs/libxkbcommon
+	x11-libs/pango"
+
+QA_PREBUILT="opt/${PN}/*"
+
+src_prepare() {
+	my_arch_suffix=$(usex amd64 x64 aarch64)
```

Comment:
Let's replace four `usex` with one `if`, something like this:
```
if use amd64; then
  my_arch_suffix=x64
  ...
  local other_arch_amd64_aarch64=aarch64
  ...
elif use arm64; then
  ...
else
  die "Unsupported arch"
fi
```

I think it looks cleaner, although I'd rather not use other-arch at all, but well, as long as there are only 2 of them it's probably ok.